docs/src/pages/demos/dividers/MiddleDividers.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Avatar from '@material-ui/core/Avatar';
+import ImageIcon from '@material-ui/icons/Image';
+import WorkIcon from '@material-ui/icons/Work';
+import BeachAccessIcon from '@material-ui/icons/BeachAccess';
+import Divider from '@material-ui/core/Divider';
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    maxWidth: 360,
+    backgroundColor: theme.palette.background.paper,
+  },
+});
+
+function MiddleDividers(props) {
+  const { classes } = props;
+  return (
+    <div className={classes.root}>
+      <List>
+        <ListItem>
+          <Avatar>
+            <ImageIcon />
+          </Avatar>
+          <ListItemText primary="Photos" secondary="Jan 9, 2014" />
+        </ListItem>
+        <Divider component="li" variant={'middle'} />
+        <ListItem>
+          <Avatar>
+            <WorkIcon />
+          </Avatar>
+          <ListItemText primary="Work" secondary="Jan 7, 2014" />
+        </ListItem>
+        <Divider variant={'middle'} component="li" />
+        <ListItem>
+          <Avatar>
+            <BeachAccessIcon />
+          </Avatar>
+          <ListItemText primary="Vacation" secondary="July 20, 2014" />
+        </ListItem>
+      </List>
+    </div>
+  );
+}
+
+MiddleDividers.propTypes = {
+  classes: PropTypes.object.isRequired,
+};
+
+export default withStyles(styles)(MiddleDividers);
```

Comment:
Could we get a demo that's closer to the recommended use of middle dividers?

<img width="880" alt="image" src="https://user-images.githubusercontent.com/357702/48444384-bc466300-e78b-11e8-8498-cf09b6ee15e1.png">

It doesn't have to be that detailed, just following the correct use case.

Reply:
Yes, I agree